src/components/SemanticModal/SemanticModal.stories.tsx

```diff
@@ -0,0 +1,155 @@
+            {
+              label: 'Primary',
+              name: 'Primary',
+              onClick: action('click-action-primary'),
+            },
+          ]}
+          message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+          title="Success"
+          onClose={() => setIsShown(false)}
+          {...args}
+        />
+      )}
+    </Padbox>
+  );
+};
+
+Playground.parameters = {
+  screenshot: { skip: true },
+};
+
+export const Success: Story<SemanticModalProps> = () => (
+  <SemanticModal
+    actions={[
+      {
+        label: 'Secondary',
+        name: 'Secondary',
+        onClick: action('click-action-secondary'),
+      },
+      {
+        label: 'Primary',
+        name: 'Primary',
+        onClick: action('click-action-primary'),
+      },
+    ]}
+    message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+    title="This is a placeholder"
+    onClose={action('close-modal')}
+  />
+);
+
+export const Warn: Story<SemanticModalProps> = () => (
+  <SemanticModal
+    actions={[
+      {
+        label: 'Secondary',
+        name: 'Secondary',
+        onClick: action('click-action-secondary'),
+      },
+      {
+        label: 'Primary',
+        name: 'Primary',
+        onClick: action('click-action-primary'),
+      },
+    ]}
+    message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+    title="This is a placeholder"
+    variant={SemanticModalVariants.warn}
+    onClose={action('close-modal')}
+  />
+);
+
+export const Error: Story<SemanticModalProps> = () => (
+  <SemanticModal
+    actions={[
+      {
+        label: 'Secondary',
+        name: 'Secondary',
+        onClick: action('click-action-secondary'),
+      },
+      {
+        label: 'Primary',
+        name: 'Primary',
+        onClick: action('click-action-primary'),
+      },
+    ]}
+    message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+    title="This is a placeholder"
+    variant={SemanticModalVariants.error}
+    onClose={action('close-modal')}
+  />
+);
+
+export const Ban: Story<SemanticModalProps> = () => (
+  <SemanticModal
+    actions={[
+      {
+        label: 'Secondary',
+        name: 'Secondary',
+        onClick: action('click-action-secondary'),
+      },
+      {
+        label: 'Primary',
+        name: 'Primary',
+        onClick: action('click-action-primary'),
+      },
+    ]}
+    message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+    title="This is a placeholder"
+    variant={SemanticModalVariants.ban}
+    onClose={action('close-modal')}
+  />
+);
```

Comment:
You can replace all this with and it will safe lots of code:
```ts
const SemanticModalTemplate: Story<SemanticModalProps> = (args) => <SemanticModal {...args} />

export const Success = SemanticModalTemplate.bind({});
Success.args = {
  actions: [
      {
        label: 'Secondary',
        name: 'Secondary',
        onClick: action('click-action-secondary'),
      },
      {
        label: 'Primary',
        name: 'Primary',
        onClick: action('click-action-primary'),
      },
  ],
  message: "You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
  title: "This is a placeholder"
  onClose: action('close-modal'),
  variant: SemanticModalVariants.success
};

export const Warn = SemanticModalTemplate.bind({});
Warn.args = {
  ...Success.args,
  variant: SemanticModalVariants.warn,
};  

export const Error = SemanticModalTemplate.bind({});
Error.args = {
  ...Success.args,
  variant: SemanticModalVariants.error,
};  

export const Ban = SemanticModalTemplate.bind({});
Ban.args = {
  ...Success.args,
  variant: SemanticModalVariants.ban,
};

Reply:
Good tip. Much shorter